### DISCUSSION TITLE
Add option to hide UI in replays

### DISCUSSION THREAD

**Discussion by crumpledmint** [ppy/osu#18846] [Ideas] [answered]
So there is a bind H in stable that allows you to hide replay interface but I couldn't find something like this in lazer. You could fold menu on the right side but it doesn't hide it completely - there are still "playback" and "visual settings" buttons

**Comment by frenzibyte** [comment, marked as answer]
You can change HUD mode to "Never" and momentarily show back using <kbd>Ctrl</kbd>. Not sure about only hiding the replays interface part, I don't see how that has any benefit.

**Reply by crumpledmint** [reply]
It looks cleaner without replay interface if you're recording replay

**Comment by monitobeko** [comment]
<kbd>Ctrl + H</kbd> hides "playback" and "visual settings" during replays in lazer.
